internal/trafficwatch/mw_request_done.go

```diff
@@ -10,6 +10,37 @@ import (
 	"github.com/signadot/cli/internal/config"
 )
 
+func waitLogged(logged, done <-chan string) chan string {
+	res := make(chan string)
+	go func() {
+		d := map[string]bool{}
+		for {
+			select {
+			case id, ok := <-logged:
+				if !ok {
+					if done == nil {
+						return
+					}
+					logged = nil
+				} else {
+					d[id] = true
+				}
+			case id, ok := <-done:
+				if !ok {
+					if logged == nil {
+						return
+					}
+					done = nil
+				} else {
+					res <- id
+					delete(d, id)
+				}
+			}
+		}
+	}()
+	return res
+}
```

Comment:
Are we supposed to use `d` within this code?

Reply:
fixed but in a merge commit due to merge conflicts
https://github.com/signadot/cli/pull/223/commits/6fd0371e45264db427fe1074594de5b4ae93bb4f